### ISSUE TITLE
Remove Strange Commands

### ISSUE THREAD

**Issue by WholeRoastedDuck** [ThunderBotDevelopment/ThunderBot#3]
Remove some interesting commands at line 1304 in /src/economy.js
 ```
case prefix+'dick':
        if (guildSettings.getGuild(message.guild.id).get('ff') == false) {
          if (message.mentions.users.first()) {
            message.channel.send("<@"+message.mentions.users.first().id+">'s dick:\n**8"+"=".repeat(getRandom(0,10))+">**");
          } else {
            message.channel.send("<@"+message.author.id+">'s dick:\n**8"+"=".repeat(getRandom(0,10))+">**");
          }
        }
          break;
        case prefix+'pp':
        if (guildSettings.getGuild(message.guild.id).get('ff') == false) {
          if (message.mentions.users.first()) {
            message.channel.send("<@"+message.mentions.users.first().id+">'s pp:\n**8"+"=".repeat(getRandom(0,10))+">**");
          } else {
            message.channel.send("<@"+message.author.id+">'s pp:\n**8"+"=".repeat(getRandom(0,10))+">**");
          }
        }
          break;
```

**Comment by WholeRoastedDuck** [comment]
#4 Pull Request to Fix

**Comment by YodaLightsabr** [comment]
I merged #4, so I’ll close this issue.
